### PR TITLE
made changes to the free trial end point

### PR DESCRIPTION
**Pull request by KlyntonREP**
called the transcribe function instead of writing it afresh

### CODE REVIEW

**Review by ColeDrain — CHANGES_REQUESTED** [review]
- Please delete freetrial.py instead of commenting it out

- And I see you check if a file is uploaded (line 230) after you copy it (line 226). This will break app if no file is uploaded.

Key points:
Check if file is not uploaded at beginning of the function
Delete freetrial.py

File: backend/app/main.py [hunk 1]
```diff
@@ -22,6 +22,9 @@
 import fastapi as _fastapi
 from auth import get_current_user
 
+import shutil
```
Comment (ColeDrain) [inline backend/app/main.py:25]: does this dependency need to be installed ?

File: backend/app/main.py [hunk 2]
```diff
@@ -216,6 +219,24 @@ async def email_verification(request: Request, token: str, db: Session = Depends
...
 
+@app.post("/tryForFree")
+async def free_trial(file: UploadFile = File(...)):
+    ####### saving the audio file
+    with open(f'{file.filename}', "wb") as buffer:
+        shutil.copyfileobj(file.file, buffer)
+    fileSize = 5242880
+    getSize = os.path.getsize(file.filename)
+    ###### transcribing the file
+    if not file:
```
Comment (ColeDrain) [inline backend/app/main.py:230]: You are checking if file is empty after copying its content

**Comment by KlyntonREP** [comment]
No boss

shutil doesn't need to be installed,  it comes wigh python

**Comment by KlyntonREP** [comment]
Ok boss, i make changes and make a new PR

**Review by DavidIkechi — APPROVED** [review]
weldone bro

**Comment by KlyntonREP** [comment]
Thanks boss

File: backend/app/main.py [hunk 3]
```diff
@@ -218,6 +221,25 @@ async def email_verification(request: Request, token: str, db: Session = Depends
...
+        raise HTTPException(status_code = 406, detail="No File Selected")
+    elif getSize > fileSize :
+        raise HTTPException(status_code = 406, detail="File Must Not Be More Than 5MB")
+    else:
+        ######### Load audio file
+        transcript = transcribe_file(file.filename)
+        transcript = transcript
+        return{"transcript": transcript}
+
+@app.get("/new_analysis/{id}", response_model=schema.Analysis, tags=['analysis'])
```
Comment (ColeDrain) [inline backend/app/main.py:242]: Great job, almost there bro, please remove this line

**Review by ColeDrain — CHANGES_REQUESTED** [review]

File: backend/app/main.py (hunk 3 above)
Comment (KlyntonREP) [reply backend/app/main.py]: Thanks Boss, i've removed it

**Review by ColeDrain — APPROVED** [review]
Great job, all issues resolved

**Pull request merged**